targets/esp32/main.c

```diff
@@ -51,7 +53,8 @@ static void espruinoTask(void *data) {
   jshInit();     // Initialize the hardware
   jswrap_ESP32_wifi_restore();
   jsvInit();     // Initialize the variables
-  vTaskDelay(1000 / portTICK_PERIOD_MS);
+  // not sure why this delay is needed?
+  vTaskDelay(200 / portTICK_PERIOD_MS);
```

Comment:
Yes - I think you are correct. I'll remove the comments then and leave only the small delay.